ambry-rest/src/main/java/com/github/ambry/rest/RestServerMain.java

```diff
@@ -50,8 +54,14 @@ public static void main(String[] args) {
               options.hardwareLayoutFilePath, options.partitionLayoutFilePath);
       clusterMap = clusterAgentsFactory.getClusterMap();
       SSLFactory sslFactory = getSSLFactoryIfRequired(verifiableProperties);
+
+      // Quota
+      QuotaConfig quotaConfig = new QuotaConfig(verifiableProperties);
+      QuotaManagerFactory quotaManagerFactory =
+          Utils.getObj(quotaConfig.quotaManagerFactory, quotaConfig, Collections.emptyList(), new MaxThrottlePolicy());
       logger.info("Bootstrapping RestServer");
-      restServer = new RestServer(verifiableProperties, clusterMap, new LoggingNotificationSystem(), sslFactory);
+      restServer = new RestServer(verifiableProperties, clusterMap, new LoggingNotificationSystem(), sslFactory,
+          quotaManagerFactory);
```

Comment:
I wouldn't create QuotaManagerFactory here in RestServerMain. because then we have to do it again in AmbryLi. 
I suggest that we create a QuotaManagerFactory in either RestServer's constructor, or even in FrontendRestRequestServiceFactory's getRestRequestService method.
If we create a QuotaManager in RestServer, then RestServer has to call init() in its start method.